### PR TITLE
Be able to send filters and aggregations for enrichment

### PR DESCRIPTION
**Pull request by dgaubert**

### CODE REVIEW

File: cartoframes/data/services/bq_datasets.py [hunk 1]
```diff
@@ -291,11 +291,13 @@ def import_dataset(self):
     def upload_dataframe(self, dataframe):
         return self._client.upload_dataframe(dataframe, self._name)
 
-    def enrichment(self, geom_type='points', variables=None, output_name=None):
+    def enrichment(self, geom_type='points', variables=None, filters=None, aggregation=None, output_name=None):
```
Comment (rafatower) [inline cartoframes/data/services/bq_datasets.py]: I'd use empty collections instead of `None` as defaults where it applies.

Comment (dgaubert) [reply cartoframes/data/services/bq_datasets.py]: I had issues with mutable default values for arguments: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Comment (dgaubert) [reply cartoframes/data/services/bq_datasets.py]: Not sure if it applies here

Comment (dgaubert) [reply cartoframes/data/services/bq_datasets.py]: Followed @oleurud tips!

**Pull request merged**